backend/app/Listeners/NotifyUsersAboutUpdatedSubmissionStatus.php

```diff
@@ -112,7 +132,7 @@ public function acceptedForReview($default)
     {
         $default['subject'] = 'Submission Accepted for Review';
         $default['body'] = $default['submission']['title'] . ' has been accepted for review. ' .
-                           'It is currently awaiting review from the assigned reviewers.';
+                           'Once it is opened for review, it will be available for review.';
```

Comment:
If this is a duplicate comment of course ignore it. We should make an issue for making these messages translatable and/or making the email templates editable by administrators